src/frontend/Sider/Personoversikt/Behandlingsoversikt/Arena/VedtaksoversiktArena.tsx

```diff
@@ -29,7 +29,7 @@ export const VedtaksoversiktArena: React.FC<{ fagsakPersonId: string }> = ({ fag
     return (
         <>
             <Heading size="small">Vedtak i Arena</Heading>
-            <DataViewer response={{ vedtakArena }}>
+            <DataViewer type={'vedtaksperioder'} response={{ vedtakArena }}>
```

Comment:
Burde vi spesifisere at det er fra Arena? 
```suggestion
            <DataViewer type={'vedtaksperioder fra Arena'} response={{ vedtakArena }}>
```